samples/LiveEffect/src/main/cpp/CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ target_link_libraries(liveEffect
         atomic
         log)
 
+# Enable optimization for release build; it could be enabled for debug build
+# if performance is of interest.
 target_compile_options(liveEffect
     PRIVATE
```

Comment:
Why is this PRIVATE? Don's suggestion was PUBLIC.
Also "CONFIG:RELEASE" instead of "CONFIG:Release". Is it case sensitive?
Don?

https://github.com/android-ndk/ndk/issues/699#issuecomment-404925125

Reply:
I don't think it matters whether it's `PUBLIC` or `PRIVATE` since this is used to define the scope of dependencies ([more here](https://stackoverflow.com/questions/26037954/cmake-target-link-libraries-interface-dependencies)). Probably best to prefer `PRIVATE`.

Reply:
It is case insensitive. Changed everything to be upper case for consistence.